pkg/types/baremetal/platform.go

```diff
@@ -61,4 +61,14 @@ type Platform struct {
 
 	// DNSVIP is the VIP to use for internal DNS communication
 	DNSVIP string `json:"dnsVIP"`
+
+	// BootstrapMachineOSImage is a URL to override the default RHCOS image
+	// for the bootstrap node.
+	// +optional
+	BootstrapMachineOSImage string `json:"bootstrapMachineOSImage,omitempty"`
+
+	// ClusterMachineOSImage is a URL to override the default RHCOS image
+	// for cluster nodes.
+	// +optional
+	ClusterMachineOSImage string `json:"clusterMachineOSImage,omitempty"`
```

Comment:
We should make use of this in kirankt/installer:pkg/asset/rhcos/image.go@bootstrap-image-override-platform-config#L64 too, right?

Reply:
Lokking for ideas here. Specifically does this belong in a new baremetal file or do we want to go about changing the openstack file which we seem to be using ATM?

Reply:
I think you could probably do it here:
https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/image.go#L84

Reply:
Added changes. Please take a look, when your time permits.

Reply:
Done. Thanks.